pkg/grpc/portergrpc/installation.go

```diff
@@ -0,0 +1,113 @@
+package portergrpc
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	iGRPC "get.porter.sh/porter/gen/proto/go/porterapis/installation/v1alpha1"
+	"get.porter.sh/porter/pkg/porter"
+	"get.porter.sh/porter/pkg/tracing"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+func makeInstOptsLabels(labels map[string]string) []string {
+	var retLabels []string
+	for k, v := range labels {
+		retLabels = append(retLabels, fmt.Sprintf("%s=%s", k, v))
+	}
+	return retLabels
+}
+
+func makeGRPCInstallation(inst porter.DisplayInstallation, gInst *iGRPC.Installation) error {
```

Comment:
given this appears to have `gInst` as an "out" parameter, I think it'd be helpful to have a small bit of documentation above this function to indicate at least that fact